### PR TITLE
Dataquery and weather changes

### PR DESCRIPTION
**Pull request by jrs65**
This supersedes: https://github.com/chime-experiment/ch_pipeline/pull/6

### CODE REVIEW

File: ch_pipeline/core/io.py [hunk 1]
```diff
@@ -190,6 +190,75 @@ def process(self):
...
+    def process(self):
+        """Load in each sidereal day.
+
+        Returns
+        -------
+        ts : andata.CorrData
+            The timestream of each sidereal day.
+        """
+
+        self.log.info(len(self.files))
```
Comment (anjakefala) [inline ch_pipeline/core/io.py:240]: Is this leftover from debugging?

File: ch_pipeline/core/io.py [hunk 2]
```diff
@@ -190,6 +190,75 @@ def process(self):
...
+        # Fetch and remove the first item in the list
+        file_ = self.files[self._file_ptr]
+        self._file_ptr += 1
+
+        # Set up a Reader class
+        rd = self._acqtype_reader[self.acqtype](file_)
+        ts = rd.read()
+
+        # Loaded file and log
+        self.log.info(f"Reading file {self._file_ptr} of {len(self.files)}. ({file_})")
```
Comment (anjakefala) [inline ch_pipeline/core/io.py:255]: Should this go before the `read()`?

File: ch_pipeline/core/dataquery.py [hunk 3]
```diff
@@ -72,12 +72,16 @@ class QueryDatabase(task.MPILoggedTask):
...
-        can be in any format ensure_unix will support, including eg
+        can be in any format ensure_unix will support, including e.g.
         20190116T150323 and 2019-1-16 08:03:23 -7
-    start_csd, end_csd : float
-        Start and end CSDs. Only used if `start_time` is not set.
-    instrument : string (default: 'chimestack')
-        data set to use
+    acqtype : string (default: 'corr')
+        Type of acquisition. Options for acqtype are: 'corr', 'hk', 'weather',
+        'rawadc', 'gain', 'flaginput', 'digitalgain'.
```
Comment (anjakefala) [inline ch_pipeline/core/dataquery.py:79]: `gain` is not supported in `LoadDataFiles`, are dataquery, and `LoadDataFiles` related?

File: ch_pipeline/core/io.py (hunk 2 above)
Comment (jrs65) [reply ch_pipeline/core/io.py:255]: Probably.

File: ch_pipeline/core/dataquery.py (hunk 3 above)
Comment (jrs65) [reply ch_pipeline/core/dataquery.py:79]: Good point. I've matched the LoadDataFiles names up to the acqtype names.

**Review by anjakefala — APPROVED** [review]

**Pull request merged**